ksqldb-engine/src/main/java/io/confluent/ksql/ServiceInfo.java

```diff
@@ -77,6 +81,7 @@ public String serviceId() {
     return serviceId;
   }
 
+  @SuppressFBWarnings(value = "EI_EXPOSE_REP", justification = "customMetricsTags is ImmutableMap")
```

Comment:
it seems like this inspection is more pain than it's worth. we'll just be disabling it all over the place - what do you think about just disabling it at the top level spotbugs config?

Reply:
The top level suppression.xml file says we should not use it, but use in-line suppression. Also, we hope that spotbug will understand `ImmutableList` etc in a future version. Thus, it might be best do just suppress inline for now.